imsim/atmPSF.py

```diff
@@ -279,16 +301,39 @@ def getPSF(self, field_pos, gsparams=None):
 
         @param [in] field position of the object relative to the boresight direction.
         """
-
         theta = (field_pos.x*galsim.arcsec, field_pos.y*galsim.arcsec)
-        psf = self.atm.makePSF(
-            self.wlen_eff,
-            aper=self.aper,
-            theta=theta,
-            t0=self.t0,
-            exptime=self.exptime,
-            gsparams=gsparams)
-        return psf
+        psfs = [
+            galsim.ChromaticAtmosphere(
+                self.atm.makePSF(
+                    self.wlen_eff,
+                    aper=self.aper,
+                    theta=theta,
+                    t0=self.t0,
+                    exptime=self.exptime,
+                    gsparams=gsparams,
+                    second_kick=False
+                ),
+                alpha=self.exponent,
+                base_wavelength=self.wlen_eff,
+                zenith_angle=0*galsim.degrees
```

Comment:
Probably a comment that this means no DCR, since we do DCR separately elsewhere.
But that raises the question, why not do DCR here now?

Reply:
> why not do DCR here now?

We probably could, but would need to parse and validate zenith/parallactic angles.  Since that's already working with the PhotonDCR operator, I'm inclined to just keep DCR there.  I do think there's a case for not applying any DCR if no zenith/parallactic/etc angles are passed to ChromaticAtmosphere in GalSim, but it currently raises if you don't pass something like zenith=0.